### ISSUE TITLE
have you reproduced the results in original paper?

### ISSUE THREAD

**Issue by iamhankai** [JunrQ/NAS#2]
have you reproduced the results in original paper?

**Comment by JunrQ** [comment]
No

**Comment by iamhankai** [comment]
> No

Me too.. How much difference now?

**Comment by JunrQ** [comment]
> > No
> 
> Me too.. How much difference now?

Actually, I didn't run experiments in original paper. The dataset, block type, arch I used for doing experiments are different.

**Comment by QiuPaul** [comment]
As you said，you only use the method of fbnet to search another task network?

**Comment by JunrQ** [comment]
> 
> 
> As you said，you only use the method of fbnet to search another task network?

I run the ImageNet search as the paper, and it converges well.

But when i do the classification task with the **sampled model**, it **failed to converge**.

I don't know why... may be different initialization? bn? random seed??

**Comment by QiuPaul** [comment]
> > As you said，you only use the method of fbnet to search another task network?
> 
> I run the ImageNet search as the paper, and it converges well.
> 
> But when i do the classification task with the **sampled model**, it **failed to converge**.
> 
> I don't know why... may be different initialization? bn? random seed??

Ah, you mean the training of the final searched network failed to converge,  or the search in another classification task?    i guess the search in another classification task failed to converge, but i also wonder how it happens, actually,i recently work on fbnet and snas...

**Comment by JunrQ** [comment]
> > > As you said，you only use the method of fbnet to search another task network?
> > 
> > 
> > I run the ImageNet search as the paper, and it converges well.
> > But when i do the classification task with the **sampled model**, it **failed to converge**.
> > I don't know why... may be different initialization? bn? random seed??
> 
> Ah, you mean the training of the final searched network failed to converge, or the search in another classification task? i guess the search in another classification task failed to converge, but i also wonder how it happens, actually,i recently work on fbnet and snas...

The search part converges ( Same settings as original paper ).

But the **network (choose one from eight blocks for every TBS(to be search) layer)** failed to converge on a very simple classification task.

One possible reason: too many layers (22 * 3 ? I don't remember the exact number), but no batchnorm.

**Comment by iamhankai** [comment]
> One possible reason: too many layers (22 * 3 ? I don't remember the exact number), but no batchnorm.

Have u tried the model provided in the paper?

**Comment by JunrQ** [comment]
> > One possible reason: too many layers (22 * 3 ? I don't remember the exact number), but no batchnorm.
> 
> Have u tried the model provided in the paper?

Not yet.

**Comment by brisker** [comment]
@JunrQ 
Python version?
PyTorch version?

**Comment by JunrQ** [comment]
> @JunrQ
> Python version?
> PyTorch version?

Python 2.7
PyTorch 1.0

**Comment by kbhardwa** [comment]
Anything new on this? I was wondering if this code reaches about 97-98% accuracy on CIFAR-10. Please let me know. Thanks a lot.
